docs_source_files/content/webdriver/http_proxies.de.md

```diff
@@ -1,23 +1,25 @@
 ---
-title: "Http proxies"
+title: "Http Proxies"
 weight: 7
 ---
 
-A proxy server acts as an intermediary for requests between a client and a server.
-In simple, the traffic flows through the proxy server on its way to the address you
-requested and back.
+Ein Proxy-Server fungiert als Zwischenstation für Anfragen zwischen
+Client und Server. Simpel erklärt werden die Daten über den Proxy zu der
+gewünschten Internetadresse und wieder retour geleitet.
 
-A proxy server for automation scripts with Selenium could be helpful for:
-- Capture network traffic
-- Mock backend calls made by the website
-- Access the required website under complex network topologies or strict
-corporate restrictions/policies.
+Ein Proxy Server für die automatisierten Seleniumskripte kann hilfreich 
```

Comment:
can we add any space or - for 'Seleniumskripte'. 
Its ok if Seleniumskripte is correct. can you please suggest on this @boris779 !

Reply:
I was also unsure about this word. I changed it to Selenium-Skripte, even if Words with dash are not common in German.